packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1030,6 +1028,7 @@ class SliverGrid extends SliverMultiBoxAdaptorWidget {
 ///    the main axis extent of each item.
 ///  * [SliverList], which does not require its children to have the same
 ///    extent in the main axis.
+@Deprecated('Use SliverLayoutBuilder instead.')
 class SliverFillViewport extends SliverMultiBoxAdaptorWidget {
```

Comment:
Can we keep this widget and just implement it in terms of SliverLayoutBuilder and SliverFixedExtendList? That would limit the scope of the API breakage and we'd only remove the RenderSliverFillViewport, I think?

Reply:
Aye